scripts/convert-gitops-template.sh

```diff
@@ -0,0 +1,105 @@
+# !/bin/bash
```

Comment:
if you've run `shellcheck` against this script, just let us (@coreydaley will I'm sure ask as well :-) ) know, otherwise, either install it on your system and run it against this script, get it clean, or use https://www.shellcheck.net/

Reply:
I just did! Thanks for the advice. Had a lot to fix indeed

Reply:
Cool.

Now using it in bash embedded in k8s YAML is too difficult, but we might want to add a github action to run on this file in the future.  Check out https://github.com/marketplace/actions/shellcheck and https://www.shellcheck.net/wiki/GitHub-Actions

@coreydaley I'm having trouble finding where we used the shellcheck github action in the past, but we did somewhere, right?

Reply:
+1 on adding this, even as a spike. At least for me was quite useful. I'll create an issue to defer the discussion there.

Reply:
Created RHDHPAI-525 to capture it.

Reply:
sounds good